src/valueobjects/ObservationType.js

```diff
@@ -0,0 +1,29 @@
+import { definitionToRecord } from '../definitions';
+export const ObservationTypeDefinition = {
+  // Metadata, filled in by http.js
+  'metadata': 'Metadata',
+
+  // Fields from the API
+  'url': null,
+  'code': null,
+  'parameter': null,
+  'unit': null,
+  'scale': null,
+  'description': null,
+  'domain_values': null,
+  'reference_frame': null,
+  'compartment': null
+};
+
```

Comment:
De keys in _ ObservationTypeDefinition_ hoeven niet als string geschreven te worden (i.t.t. Python)